frontend/src/components/home/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Sidebar from '../sidebar';
+import homeStyles from './home.module.scss';
+
+const Home = () => (
+  <>
+    <div className={homeStyles.sidebar}>
+      <Sidebar />
+    </div>
+    <div />
+  </>
```

Comment:
Just curious, what's the purpose of empty tags and the empty div>

Reply:
following the ticket directly which says to create "container" for the the blank part of home

Reply:
the empty tag is a way to stop react from complaining and saying you should only return one react child (it adds nothing to the dom)

Reply:
Yeah, react complains if you don't.  rather than an empty tag I would maybe wrap all the contents in a div with a className something like home-container?

Reply:
For now ive just removed it because it's cleaner and we can add things back in when we need to.